src/main/java/seedu/notus/ui/Formatter.java

```diff
@@ -112,14 +112,14 @@ public static String formatNotes(String header, ArrayList<Note> notes, Notebook
     //@@author Nazryl
     public static String formatNote(String message, Note note) {
         String formattedString = "";
-        String header = message.concat(note.getTitle() + " " + note.getTagsName().trim());
+        String header = message.concat(note.getTitle() + " " + note.getTagsName());
 
         if (note.getPinned()) {
-            header = header.concat(" | ISPIN: " + note.getPinnedString());
+            header = header.concat("| ISPIN: " + note.getPinnedString() + " ");
         }
 
         if (note.getIsArchived()) {
-            header = header.concat(" | ISARCHIVE: " + note.getIsArchived());
+            header = header.concat("| ISARCHIVE: " + note.getIsArchived());
```

Comment:
Just to confirm, getPinnedString returns a string value and getIsArchived returns a boolean value. Just raising this to confirm your intention was to use getPinnedString and not getIsPinned? 👍

Reply:
Yes confirm. Open to changes. Both relay the same message to let the user know its pinned and archived.

Reply:
In that case, might be better to standardize to have both returning the boolean value.